Sites/styles.css

```
@@ -0,0 +1,80 @@
+body {
+    @media (max width: 1280px) @media (min-width: 375px) height: 4600px;
+    background: linear-gradient(#ebf0fa, #adc2eb);
+}
+
+h1 {
+    font-family: Bodoni;
```

Need to include some backup fonts here for folks who don't have Bodoni on their machines (according to [this](https://www.cssfontstack.com/Bodoni-MT), Bodoni is on a little over half of Windows machines and on zero Macs by default). When all else fails, consider searching for your font name + "font stack" to see what other things folks have used, e.g.:
```suggestion
    font-family: Bodoni, "Bodoni MT", Didot, "Didot LT STD", "Hoefler Text", Garamond, "Times New Roman", serif;
```